docs/app-design.md

```diff
@@ -0,0 +1,148 @@
+* They also fall into 3 possible categories: Online homework, Adaptive courseware and Customization tools
+* The Partner data is pulled from Salesforce. See [Salesforce section](#Salesforce)
+
+### API
+* APIs for various data
+  * images - images used on OS site - is this correct? Cloudfront?
+  * documents - PDFs, PowerPoints, text files, etc.
+  * adopters - is this correct? cloudfront?
+  * progress - what is this?
+  * sticky note - a time limited banner that displays at the top of the site
+  * schools - list of schools using OpenStax books. Used for map display on OS site. Data comes from Salesforce
+  * mapbox - data for mapbox. Data comes from Salesforce
+  * flags - not sure what this is for
+  * errata-fields - errata data
+  * footer - data for site footer
+
+### Books
+* APIs and models for Book information
+* API allows getting information for all books or a single book
+
+### Duplicate Books
+* Form that, once completed, allows for the duplication of book data into a new book model
+* Uses Wagtail import-export package
+
+### Errata
+* seperate Django app that is used for handling errata through a workflow. The app does not use Wagtail as does the rest of OS CMS.
+* Uses standard Django templates for the user interface
+* The workflow is a series of states
+  * New
+  * Editorial review
+  * Reviewed
+  * Archived
+  * Completed
+* Errata is entered via a form on the OS site. Users can choose to be updated via email on the status of their entry. Emails are sent as the status changes.
+* Users can also view errata on the OS site. Data is available via an API.
+* Errata can have various resolutions
+  * Duplicate
+  * Not An Error
+  * Will Not Fix
+  * Approved
+  * Major Contains code for the BlogBook Revision
+  * Technical Error
+  * Sent to Customer Support
+  * More Information Requested
+* Errata can be various error types
+  * Other factual inaccuracy in content
+  * General/pedagogical suggestion or question
+  * Incorrect answer, calculation, or solution
+  * Broken link
+  * Typo
+  * Other
+* Emails are sent to users who entered the errata as it moves through the workflow. users can opt out of receiving the emails.
+* Errata can be filtered in the UI
+
+### Extra Admin Filters
+* Used to filter errata
+* Allows users to filter multiple books in the same query
+
+### Global Settings
+* Views and models for items on the Admin Settings menu
+  * Sticky Note
+  * Footer
+  * Cloudfront Distribution 
+
+### Mail
+* Mainly used to send the redirect report email to the Content team
+* There could be some dead code in this app
+
+### News
+* Contains code for the Blog Atom feed
+* Contains code for blog posts
+* Has APIs for a full list of blog posts or an single blog post
+* Contains code for OS press kit
+* Search is implemented for blog posts
+
+### OpenStax
+* The application used for settings and general items
+* APIs for accessing Documents and Images
+* Document API checks if faculty documents should be hidden or not
```

Comment:
This is one item that uses a feature flag.

Reply:
@mwharrison This has been added